nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -640,6 +674,16 @@ in {
         ++ (optional (versionOlder cfg.package.version "23") (upgradeWarning 22 "22.05"))
         ++ (optional (versionOlder cfg.package.version "24") (upgradeWarning 23 "22.05"))
         ++ (optional (versionOlder cfg.package.version "25") (upgradeWarning 24 "22.11"))
+        ++ (optional cfg.enableBrokenCiphersForSSE ''
+          You enabled OpenSSL 1.x for NextCloud, this is not necessary except if you
+          are running a RC4-based server side encryption, we advise you to disable server-side
```

Comment:
Hmm... with respect to the discussion below I'm not sure if that's a bit too harsh. How about something like "You're using PHP's openssl extension built against OpenSSL 1.1. This is only necessary if you're using server-side encryption. Please keep in mind that it's using the broken RC4 cipher"?

Reply:
That's a lot better than my version :) thank you!